cime_config/usermods_dirs/NEON/defaults/user_nl_clm

```diff
@@ -19,7 +19,7 @@
 !----------------------------------------------------------------------------------
 
 flanduse_timeseries = ' '   ! This isn't needed for a non transient case, but will be once we start using transient compsets
-fsurdat = "$DIN_LOC_ROOT/lnd/clm2/surfdata_map/NEON/surfdata_hist_78pfts_CMIP6_simyr2000_${NEONSITE}_c211102.nc"
+fsurdat = "$DIN_LOC_ROOT/lnd/clm2/surfdata_map/NEON/surfdata_0.9x1.25_hist_78pfts_CMIP6_simyr2000_${NEONSITE}_c221014.nc"
```

Comment:
@wwieder it's confusing to have the global resolution of 0.x1.25 in the filename. It's also maybe not good to not mention the resolution other than the NEON site name. So maybe it should be something like...

surfdata_1x1_CLMUSR_DAT_hist_78pfts_CMIP6_simyr2000_${NEONSITE}_c221014.nc

Reply:
Or you could put the NEONSITE near the beginning...

surfdata_1x1_${NEONSITE}_hist_78pfts_CMIP6_simyr2000_c221014.nc

That would be more similar to the other supported single point datasets.

Reply:
We like the second one

Reply:
We should change this in subset_data

Reply:
I'll do this

Reply:
@ekluzek when you update the names of these surface datasets and upload them in input data, will you also modify `usermods_dirs/NEON/defaults/user_nl_clm` appropriately?

Reply:
Yes, I can do that.

Reply:
My only thought here is that we also want naming with subset_data to work fine for NON-NEON sites, or generic single point cases...

Reply:
Yep, we handled this. And it changes subset_data, so it'll work the same way.